### PR TITLE
Replace all mentions of Slack with Matrix

### PR DESCRIPTION
**Pull request by rileyjshaw**
Screenshot of /about page:

matrix

@mbransn, the stroke weight of the provided SVG seems inconsistent vs the other icons on the /about page. Is that okay with you?

### CODE REVIEW

File: contribute.json [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
-        "url": "https://github.com/mozilla/voice-web",
-        "license": "MPL2",
-        "tests": "https://travis-ci.org/mozilla/voice-web/"
-    },
-    "participate": {
-        "home": "https://voice.mozilla.org",
-        "docs":"https://github.com/mozilla/voice-web/blob/master/CONTRIBUTING.md",
-        "forum": "https://discourse.mozilla.org/c/voice",
-        "slack": "https://common-voice.slack.com/",
-        "slack-invite": "https://common-voice-slack-invite.herokuapp.com/"
```
Comment (rileyjshaw) [inline contribute.json]: This file was re-indented by our formatting rules. These were the only lines actually removed.

File: contribute.json [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
+  "repository": {
+    "url": "https://github.com/mozilla/voice-web",
+    "license": "MPL2",
+    "tests": "https://travis-ci.org/mozilla/voice-web/"
+  },
+  "participate": {
+    "home": "https://voice.mozilla.org",
+    "docs": "https://github.com/mozilla/voice-web/blob/master/CONTRIBUTING.md",
+    "forum": "https://discourse.mozilla.org/c/voice",
+    "matrix": "https://chat.mozilla.org/#/room/#common-voice:mozilla.org"
```
Comment (rileyjshaw) [inline contribute.json]: This file was re-indented by our formatting rules. This was the only line actually added.

**Comment by mbransn** [comment]
> @mbransn, the stroke weight of the provided SVG seems inconsistent vs the other icons on the /about page. Is that okay with you?

@rileyjshaw agreed it is inconsistent, it was pulled from Matrix so not part of our icon family. I'm concerned that retroactively increasing stroke weight would cause a different suite of issues. The icon also appears grey and should be full white. In look at this our Discourse icon is also not aligned with how it appears elsewhere at Mozilla. 

Here's an updated approach to this that will match the chat icons we use elsewhere at Mozilla, let me know thoughts:

- Update Discourse icon to this (in white): https://www.figma.com/file/rtFgmg8m42n4Be17ldPa7Z/Master-Library?node-id=0%3A1454
- Update Matrix icon to this (in white): https://www.figma.com/file/rtFgmg8m42n4Be17ldPa7Z/Master-Library?node-id=0%3A1456
- Reorder icons so they match the text above: email, discourse, github, matrix
- Remove break after first sentence in copy block

**Comment by rileyjshaw** [comment]
- [x] Replace Matrix icon on /about page
- [x] Replace Discourse icon on /about page
- [x] Reorder icons on /about page
- [x] Remove line break on /about page
- [x] Replace Discourse icon in page footer

@mbransn here's an updated screenshot:

icons

I also updated the Discourse icon in the homepage footer for consistency, let me know if you want me to roll that back. Screenshot:

Screen Shot 2020-02-24 at 1 51 04 PM

**Comment by mbransn** [comment]
> Replace Discourse icon in page footer 💯 

You're a rockstar.

Looks and interacts wonderfully, thanks so much @rileyjshaw. 👏

**Comment by rileyjshaw** [comment]
Sweet!

**Pull request merged**